qiskit/visualization/state_visualization.py

```diff
@@ -684,27 +684,11 @@ def plot_state_qsphere(state, figsize=None, ax=None, show_state_labels=True,
         raise ImportError('Must have Matplotlib installed. To install, run '
                           '"pip install matplotlib".')
 
-    from mpl_toolkits.mplot3d import proj3d
-    from matplotlib.patches import FancyArrowPatch
     import matplotlib.gridspec as gridspec
     from matplotlib import pyplot as plt
     from matplotlib.patches import Circle
     from matplotlib import get_backend
-
-    class Arrow3D(FancyArrowPatch):
-        """Standard 3D arrow."""
-
-        def __init__(self, xs, ys, zs, *args, **kwargs):
-            """Create arrow."""
-            FancyArrowPatch.__init__(self, (0, 0), (0, 0), *args, **kwargs)
-            self._verts3d = xs, ys, zs
-
-        def draw(self, renderer):
-            """Draw the arrow."""
-            xs3d, ys3d, zs3d = self._verts3d
-            xs, ys, _ = proj3d.proj_transform(xs3d, ys3d, zs3d, renderer.M)
-            self.set_positions((xs[0], ys[0]), (xs[1], ys[1]))
-            FancyArrowPatch.draw(self, renderer)
+    from visualization.bloch import Arrow3D
```

Comment:
```suggestion
    from qiskit.visualization.bloch import Arrow3D
```
I think this should fix the import issues 😄

Reply:
Thanks for the insight, Abby. But it does not seem to pass the test. Instead, it shows errors while running _make lint_ and _make test_ (Note: The error is not produced by the changes that were made by me). It would be helpful if someone can tell me why this happens.

Reply:
I pulled your PR and ran make test and make lint with the change I suggested and I'm not getting any errors 😄  You might need to check you have the right requirements installed (you can double check you've installed from source correctly [here](https://qiskit.org/documentation/contributing_to_qiskit.html#installing-terra-from-source)). You could also just accept the change here in GitHub and see if the CI checks pass

Reply:
That sounds great. But, I had used the exact same link to install the requirements. I'll create a new environment and start it from scratch. Anyways thanks for the update.

Reply:
I reinstalled the dependencies on a different environment and voila it worked! Thanks again at @javabster and @1ucian0 for your time.